Add changelog for #7288 (#7290)
firebase/firebase-ios-sdk: Firestore/CHANGELOG.md

# Unreleased

> - [fixed] Fixed an issue where using `FieldValue.delete()` would only delete the

Reviewer: ```suggestion
- [fixed] Fixed an issue where using `FieldValue.arrayRemove()` would only delete the
```

>   first occurrence of the element in the document array in latency

Reviewer: ```suggestion
  first occurrence of an element in an array in a latency
```
